### ISSUE TITLE
All relevant images should have alt useful attributes

### ISSUE THREAD

**Issue by leeenglestone** [dotnet-foundation/website#117]
For Web Accessibility, all relevant images on the site should have useful alt attributes.

Some images are currently missing them

![image](https://user-images.githubusercontent.com/2360725/84247978-b730cd80-ab00-11ea-8d46-7908f7b198a0.png)
